paths.c

```diff
@@ -370,13 +370,20 @@ void pv_paths_lib_lxc_lxcpath(char *buf, size_t size)
 }
 
 #define PV_ETC_PATHF "%s/%s"
+#define PV_ETC_POLICY_PATHF "%s/pantavisor/policy/%s"
 
 void pv_paths_etc_file(char *buf, size_t size, const char *name)
 {
 	SNPRINTF_WTRUNC(buf, size, PV_ETC_PATHF, pv_config_get_system_etcdir(),
```

Comment:
lets normalize the filename so someone passing in ../../some/other/unexpected/path wont be able to trick the parser trying to load files that were not intended to be used as policies.

one idea is to send the name through a mangler that replaces all "/" with "-"

Reply:
this has further implications. There are a bunch of paths that can be configured in pantavisor.config and/or any other policy config file already (almost every path in pantavisor is configurable since appengine). Should we shield those too? and how?